assets/src/block-validation/error/error-content.js

```diff
@@ -0,0 +1,232 @@
+					__( `%s (theme)`, 'amp' ),
+					blockSource.title,
+				);
+				break;
+
+			default:
+				source = blockSource?.title || getErrorSourceTitle( sources );
+				break;
+		}
+	} else {
+		source = getErrorSourceTitle( sources );
+	}
+
+	if ( ! source ) {
+		source = __( 'Unknown', 'amp' );
+	}
+
+	return (
+		<>
+			<dt>
+				{ __( 'Source', 'amp' ) }
+			</dt>
+			<dd>
+				{ source }
+			</dd>
+		</>
+	);
+}
+ErrorSource.propTypes = {
+	blockTypeName: PropTypes.string,
+	clientId: PropTypes.string,
+	sources: PropTypes.arrayOf( PropTypes.object ),
+};
+
+/**
+ * @param {Object} props
+ * @param {number} props.status Error status.
+ */
+function MarkupStatus( { status } ) {
+	let keptRemoved;
+	if ( [ VALIDATION_ERROR_NEW_ACCEPTED_STATUS, VALIDATION_ERROR_ACK_ACCEPTED_STATUS ].includes( status ) ) {
+		keptRemoved = (
+			<span className="amp-error__kept-removed amp-error__kept-removed--removed">
+				{ __( 'Removed', 'amp' ) }
+				<span>
+					<AMPDelete />
+				</span>
+			</span>
+		);
+	} else {
+		keptRemoved = (
+			<span className="amp-error__kept-removed amp-error__kept-removed--kept">
+				{ __( 'Kept', 'amp' ) }
+				<span>
+					<AMPAlert />
+				</span>
+			</span>
+		);
+	}
+
+	let reviewed;
+	if ( [ VALIDATION_ERROR_ACK_ACCEPTED_STATUS, VALIDATION_ERROR_ACK_REJECTED_STATUS ].includes( status ) ) {
+		reviewed = __( 'Yes', 'amp' );
+	} else {
+		reviewed = __( 'No', 'amp' );
+	}
+
+	return (
+		<>
+			<dt>
+				{ __( 'Markup status', 'amp' ) }
+			</dt>
+			<dd>
+				{ keptRemoved }
+			</dd>
+			<dt>
+				{ __( 'Reviewed', 'amp' ) }
+			</dt>
+			<dd>
+				{ reviewed }
+			</dd>
+		</>
+	);
+}
+MarkupStatus.propTypes = {
+	status: PropTypes.number,
+};
+
+/**
+ * @param {Object} props
+ * @param {string} props.blockTypeTitle Title of the block type.
+ * @param {string} props.clientId Block ID.
+ */
+function BlockType( { blockTypeTitle, clientId } ) {
+	if ( clientId ) {
+		return (
+			<>
+				<dt>
+					{ __( 'Block type', 'amp' ) }
+				</dt>
+				<dd>
+					<span className="amp-error__block-type-description">
+						{ blockTypeTitle || __( 'unknown', 'amp' ) }
+					</span>
+				</dd>
+			</>
+		);
+	}
+
+	return null;
+}
+BlockType.propTypes = {
+	blockTypeTitle: PropTypes.string,
+	clientId: PropTypes.string,
+};
+
+/**
+ * Content inside an error panel.
+ *
+ * @param {Object} props Component props.
+ * @param {Object} props.blockType Block type details.
+ * @param {string} props.clientId Block client ID
+ * @param {number} props.status Number indicating the error status.
+ * @param {string} props.title Error title.
+ * @param {Object} props.error Error details.
+ * @param {Object[]} props.error.sources Sources from the PHP backtrace for the error.
+ */
+export function ErrorContent( { blockType, clientId, status, title, error: { sources } } ) {
+	const blockTypeTitle = blockType?.title;
+	const blockTypeName = blockType?.name;
+
+	const [ titleText, nodeName ] = title.split( ':' ).map( ( item ) => item.trim() );
```

Comment:
Splitting by `:` seems to put too much reliance on the string formatting returned by `\AMP_Validation_Error_Taxonomy::get_error_title_from_code()`. That method will need to be overhauled to allow for more structured data. This may be required for the design proposed in https://github.com/ampproject/amp-wp/issues/5697#issuecomment-749176913.

Reply:
I did try to account for the unreliability of using the colon. If there is no colon in the string, then splitting by `:` means `titleText` is the entire string, and `nodeName` is undefined. Just below, the line relying on these two string segments is only rendered if `nodeName` is not empty. It will be OK if it doesn't render because the full title will be shown in the PanelBody header. 

I'd suggest this works for now but agree that this string manipulation isn't great and we need more structured data in the next iteration.

Reply:
Agreed. It works for now, but it's an example of something where we will want to improve later.

```suggestion
	// @todo Refactor AMP_Validation_Error_Taxonomy::get_error_title_from_code() to return structured data.
	const [ titleText, nodeName ] = title.split( ':' ).map( ( item ) => item.trim() );
```